debian/changelog

```diff
@@ -1,3 +1,9 @@
+daos (2.7.101-9) unstable; urgency=medium
+  [ Samirkumar Raval ]
+  * Changing the default log location to /var/log/daos from /tmp
+
+ -- Samirkumar Raval <samirkumar.raval@hpe.com>  Thu, 4 Apr 2025 11:55:00 +0000
```

Comment:
The change dates should be synchronized between the `utils/rpms/daos.spec` and `debian/changelog` files - also Apr 4th was a Friday.

Reply:
Done